docs/ocf-ble.md

```diff
@@ -82,44 +97,62 @@ seconds for it to appear.
-HTTP: 200
-$ ./oic-get /a/light?di=31bf6309-8ebf-4309-5fbf-630930c06309
-10.237.72.146:80 /a/light?di=31bf6309-8ebf-4309-5fbf-630930c06309
-{"properties":{"state":false,"power":10}}
-HTTP: 200
-$ ./oic-get /a/light?di=31bf6309-8ebf-4309-5fbf-630930c06309
-10.237.72.146:80 /a/light?di=31bf6309-8ebf-4309-5fbf-630930c06309
-{"properties":{"state":true,"power":10}}
-HTTP: 200
-$ ./oic-get /a/light?di=31bf6309-8ebf-4309-5fbf-630930c06309
-10.237.72.146:80 /a/light?di=31bf6309-8ebf-4309-5fbf-630930c06309
-{"properties":{"state":false,"power":10}}
-HTTP: 200
-$ ./oic-get /a/light?di=31bf6309-8ebf-4309-5fbf-630930c06309
-10.237.72.146:80 /a/light?di=31bf6309-8ebf-4309-5fbf-630930c06309
-{"properties":{"state":true,"power":10}}
-HTTP: 200
-As you can see, the discovery and retrieve resources works.
-```
+2. Test script method
+
+  Run the oic-get script from the iot-rest-api-server test/ directory.
+
+  ```
+  $ ./oic-get /res
+  10.237.72.146:80 /res
+  [{"di":"b492c1dd-fa51-47f1-a03c-d31495a0b99a","links":[{"href":"/oic/sec/doxm","
+  rt":"oic.r.doxm","if":"oic.if.baseline"}]},{"di":"b492c1dd-fa51-47f1-a03c-d31495
+  a0b99a","links":[{"href":"/oic/sec/pstat","rt":"oic.r.pstat","if":"oic.if.baseli
+  ne"}]},{"di":"b492c1dd-fa51-47f1-a03c-d31495a0b99a","links":[{"href":"/oic/d","r
+  t":"oic.wk.d","if":"oic.if.baseline"}]},{"di":"b492c1dd-fa51-47f1-a03c-d31495a0b
+  99a","links":[{"href":"/oic/p","rt":"oic.wk.p","if":"oic.if.baseline"}]},{"di":"
+  31bf6309-8ebf-4309-5fbf-630930c06309","links":[{"href":"/oic/p","rt":"oic.wk.p",
+  "if":"oic.if.r"}]},{"di":"31bf6309-8ebf-4309-5fbf-630930c06309","links":[{"href"
+  :"/oic/d","rt":"oic.d.zjs","if":"oic.if.r"}]},{"di":"31bf6309-8ebf-4309-5fbf-630
+  930c06309","links":[{"href":"/a/light","rt":"core.light","if":"oic.if.rw"}]}]
+  HTTP: 200
+  $ ./oic-get /a/light?di=31bf6309-8ebf-4309-5fbf-630930c06309
+  10.237.72.146:80 /a/light?di=31bf6309-8ebf-4309-5fbf-630930c06309
+  {"properties":{"state":false,"power":10}}
+  HTTP: 200
+  $ ./oic-get /a/light?di=31bf6309-8ebf-4309-5fbf-630930c06309
+  10.237.72.146:80 /a/light?di=31bf6309-8ebf-4309-5fbf-630930c06309
+  {"properties":{"state":true,"power":10}}
+  HTTP: 200
+  $ ./oic-get /a/light?di=31bf6309-8ebf-4309-5fbf-630930c06309
+  10.237.72.146:80 /a/light?di=31bf6309-8ebf-4309-5fbf-630930c06309
+  {"properties":{"state":false,"power":10}}
+  HTTP: 200
+  $ ./oic-get /a/light?di=31bf6309-8ebf-4309-5fbf-630930c06309
+  10.237.72.146:80 /a/light?di=31bf6309-8ebf-4309-5fbf-630930c06309
+  {"properties":{"state":true,"power":10}}
+  HTTP: 200
+  ```
```

Comment:
OcfServer.js automatically toggles the output on every GET, so we did not need to show it 4x. This is much better.